example/ios/Podfile.lock

```diff
@@ -264,20 +264,20 @@ PODS:
     - React-Core
     - React-RCTImage
     - React-RCTText
-  - RNCAsyncStorage (1.15.2):
+  - RNCAsyncStorage (1.15.5):
     - React-Core
   - RNCMaskedView (0.1.10):
     - React
   - RNGestureHandler (1.10.3):
     - React-Core
-  - RNReanimated (1.13.2):
+  - RNReanimated (1.13.3):
     - React-Core
-  - RNScreens (2.17.1):
+  - RNScreens (2.18.1):
     - React-Core
   - Yoga (1.14.0)
 
 DEPENDENCIES:
-  - "DatadogSDKReactNative (from `../node_modules/@datadog/mobile-react-native`)"
+  - DatadogSDKReactNative (from `../../packages/core`)
```

Comment:
Are changes in this file coming from `pod install`/`pod update`? When I try to run these commands changes in this file (3rd party versions, location of DD SDK) are reverted back, only change of bridge version is kept (but not location).

AFAIK versions should come form versions defined in the `yarn.lock`, and for example `react-native-screens` has version `2.17.1` there.

Reply:
This came from pod update indeed

Reply:
Can you try to remove your `example/node_modules` folder and do `yarn example install` again? After that `pod install`/`pod update` should be good (versions are taken from `.podspec` files in the `node_modules`). I think I had versions as in this change when I tried RN `0.64.x` and if I remember correctly I had the following compilation issue https://github.com/software-mansion/react-native-screens/issues/842 with `RNScreens 2.18.1`